### DISCUSSION TITLE
segmentation fault while specifying an Experimental Cell Size Distribution

### DISCUSSION THREAD

**Discussion by vishalnp37** [neperfepx/neper#713] [Q&A] [answered]
HI I am trying to simulate a microstructure of melt and while using Laguerre tessellation ,as a sample, I created a .txt file called mydistrib.txt which contains
 0.2 10.0
0.3 20.0
0.4  5.0
when I try to run :
```
neper -T -n from_morpho -morpho "diameq:custom(mydistrib),1-sphericity:lognormal(0.145,0.03)" -statcell diameq -o mytess
```

It is giving me a segmentation fault
![image](https://github.com/neperfepx/neper/assets/128611901/326d3294-8448-4e0b-92a8-61f03a3e4590)
![image](https://github.com/neperfepx/neper/assets/128611901/6d361db3-b08c-43f1-8ab8-efe80c620f1c)

**Comment by rquey** [comment, marked as answer]
Use the full file name: `diameq:custom(mydistrib.txt)`, not just `diameq:custom(mydistrib)`.
